app/parsing_scripts/git_update.py

```diff
@@ -49,8 +49,8 @@ def get_list_of_changed_ports(new_hash=False, old_hash=False, root=BASE_DIR):
             if old_hash is False:
                 try:
                     # Try to fetch the most recent hash from database
-                    old_hash_object = Commit.objects.all().order_by('-updated_at').first()
-                    old_hash = old_hash_object.hash
+                    old_hash_object = LastPortIndexUpdate.objects.all().first()
+                    old_hash = old_hash_object.git_commit_hash
                 except AttributeError:
                     # If database is empty, use the current HEAD
```

Comment:
This logic kind of doesn't make sense. The old hash would be the latest commit? If anything, I assume you should start from the first commit instead :)

Reply:
`git pull` runs after this step. So the commit before `git pull` is the `old_hash` and the commit after `git pull` is the new hash. I am sorry, I could not get what do you mean by first commit?

This happens only in that case when the database is empty all well as a commit is not supplied.

Reply:
If the database is empty, it could be that no ports exist in the database, to start with. So you could start with commit 1 in the git history, or basically the equivalent of updating all ports :)

I clearly missed the fact that this happens before you fetch contents from git. It's still not optimal, but well ...

Reply:
Okay, thank you. Using the first commit makes sense.